Auth0/CredentialsManager.swift

```diff
@@ -233,21 +231,12 @@ public struct CredentialsManager {
     }
 
     func willExpire(_ credentials: Credentials, within ttl: Int) -> Bool {
-        if let expiresIn = credentials.expiresIn {
-            return expiresIn < Date(timeIntervalSinceNow: TimeInterval(ttl))
-        }
-
-        return false
+        return credentials.expiresIn < Date(timeIntervalSinceNow: TimeInterval(ttl))
     }
 
     func hasExpired(_ credentials: Credentials) -> Bool {
-        if let expiresIn = credentials.expiresIn {
-            if expiresIn < Date() { return true }
-        }
-
-        if let token = credentials.idToken, let jwt = try? decode(jwt: token) {
-            return jwt.expired
-        }
+        if credentials.expiresIn < Date() { return true }
+        if let jwt = try? decode(jwt: credentials.idToken) { return jwt.expired }
```

Comment:
@adamjmcgrath should we keep taking into account the ID Token expiration to determine the validity of the credentials?

Reply:
Yep 👍  - we should only use the ID Token `exp` when validating the ID Token

Reply:
Ok, I'll remove that check then